php/libraries/LorisForm.class.inc

```diff
@@ -1352,11 +1352,11 @@ class LorisForm
      *
      * @param callable $func The callback to use to process this form.
      *
-     * @return none
+     * @return mixed Preferably `bool`...
```

Comment:
I am unsure if phan can actually statically analyze for this. We call this by passing an `array` (with the class name and method name as elements, I think?) and not a `function`.

So, I'm not sure if phan is good enough at following something like that. I doubt it. Also, nothing really stops the user from making the func return an int. So, I assumed someone might want to leverage that or something.

I have my `is_bool()` checks elsewhere, because of that.